http_get.js

```diff
@@ -0,0 +1,40 @@
+import http from "k6/http";
+const SONG_MAX_COUNT = 10000000; // 10 million
+
+export let options = {
+  vus: 700,
+  duration: "45s"
```

Comment:
I remember you said you were using multi-minute tests earlier -- why the switch?

Reply:
When I did 5min tests? It was taking so long, so I cut down to 1m or 30s for most tests